src/main/java/com/code_intelligence/jazzer/driver/FuzzTargetRunner.java

```diff
@@ -166,7 +170,21 @@ private static int runOne(long dataPtr, int dataLength) {
       //  1. Let libFuzzer pass in a null pointer if the byte array hasn't changed since the last
       //     call to our custom mutator and skip the read entirely.
       //  2. Implement a InputStream backed by Unsafe to avoid the copyToArray overhead.
-      mutator.read(new ByteArrayInputStream(copyToArray(dataPtr, dataLength)));
+      byte[] buf = copyToArray(dataPtr, dataLength);
+      boolean readCompletely = mutator.read(new ByteArrayInputStream(buf));
```

Comment:
I haven't looked into the implementation yet, but is this `readCompletely` or rather `readExactly`?